PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetsEEC.h

```diff
@@ -125,66 +125,66 @@ Bool_t fDoFillEncMC;      ///< to fill the matched mc plane
+  TH3D *E3C_det_pt_hist_3d; ///< initializing 3d histogram with det level E3C and det and true jet pt
+  TH3D *E3C_tru_pt_hist_3d; ///< initializing 3d histogram with tru level E3C and det and true jet pt
   
-  TH3D *E3C_det_pt_hist_log_3d;
-  TH3D *E3C_tru_pt_hist_log_3d;
+  TH3D *E3C_det_pt_hist_log_3d; ///< initializing 3d histogram with det level E3C and det and true jet pt - log scale
+  TH3D *E3C_tru_pt_hist_log_3d; ///< initializing 3d histogram with tru level E3C and det and true jet pt - log scale
 
-  TH3D *N2_det_pt_hist_3d;
-  TH3D *N2_tru_pt_hist_3d;
+  TH3D *N2_det_pt_hist_3d; ///< initializing 3d histogram with det level EEC num pairs and det and true jet pt
+  TH3D *N2_tru_pt_hist_3d; ///< initializing 3d histogram with tru level EEC num pairs and det and true jet pt
   
-  TH3D *N2_det_pt_hist_log_3d;
-  TH3D *N2_tru_pt_hist_log_3d;
+  TH3D *N2_det_pt_hist_log_3d; ///< initializing 3d histogram with det level EEC num pairs and det and true jet pt - log scale
+  TH3D *N2_tru_pt_hist_log_3d; ///< initializing 3d histogram with tru level EEC num pairs and det and true jet pt - log scale
   
-  TH3D *N3_det_pt_hist_3d;
-  TH3D *N3_tru_pt_hist_3d;
+  TH3D *N3_det_pt_hist_3d;  ///< initializing 3d histogram with det level E3C num pairs and det and true jet pt
+  TH3D *N3_tru_pt_hist_3d; ///< initializing 3d histogram with tru level E3C num pairs and det and true jet pt
   
-  TH2D *EEC_det_match_pt_det;
-  TH2D *EEC_tru_match_pt_tru;
+  TH2D *EEC_det_match_pt_det; ///< initializing 2D histogram of det level EEC and det jet pt
+  TH2D *EEC_tru_match_pt_tru;  ///< initializing 2D histogram of tru level EEC and tru jet pt
   
-  TH2D *E3C_det_match_pt_det;
-  TH2D *E3C_tru_match_pt_tru;
+  TH2D *E3C_det_match_pt_det; ///< initializing 2D histogram of det level E3C and det jet pt
+  TH2D *E3C_tru_match_pt_tru; ///< initializing 2D histogram of tru level E3C and tru jet pt
   
-  TH2D *EEC_det_match_pt_det_log;
-  TH2D *EEC_tru_match_pt_tru_log;
+  TH2D *EEC_det_match_pt_det_log; ///< initializing 2D histogram of det level EEC and det jet pt - log scale
+  TH2D *EEC_tru_match_pt_tru_log; ///< initializing 2D histogram of tru level EEC and tru jet pt - log scale
   
-  TH2D *E3C_det_match_pt_det_log;
-  TH2D *E3C_tru_match_pt_tru_log;
+  TH2D *E3C_det_match_pt_det_log; ///< initializing 2D histogram of det level E3C and det jet pt - log scale
+  TH2D *E3C_tru_match_pt_tru_log; ///< initializing 2D histogram of tru level E3C and tru jet pt - log scale
   
-  TH1D *pt_tru; //all tru level
-  TH1D *pt_tru_match; //tru level matched
-  TH1D *pt_det; //all det level
-  TH1D *pt_det_match; //det level matched
+  TH1D *pt_tru; ///< all tru level
+  TH1D *pt_tru_match; ///< tru level matched
+  TH1D *pt_det; ///< all det level
+  TH1D *pt_det_match; ///< det level matched
   
-  TH1D *test_hist; //test histogram
+  TH1D *test_hist; ///< test histogram
   
-  TH2D *R_matrix;
+  TH2D *R_matrix; ///< response matrix for jet pt
```

Comment:
Are you sure that you want to stream the hist pointers? Streaming hist pointers is often more trouble than it's worth, especially since they reinitialize just fine. To be concrete, I would suggest `//!<!` for your hist pointers unless you have a particular reason

Reply:
Ah I see. No, there isn't a particular reason (I was following what I saw in other analyses). I will make the changes and recommit. Thanks for the suggestion!

Reply:
Sorry, for some reason the changes didn't commit. I'll try again.

Reply:
Re-committed with the suggested changes.